form_style.css

```diff
@@ -0,0 +1,63 @@
+h2{
+    color:ghostwhite;
+    text-align: center;
+    font-size: 50px;
+    }
+    
+    h1{
+        font-size: 90px;
+        color:ghostwhite;
+        text-align: center;
+    }
+    
+    p {
+        color: #fbeec1;
+        font-size: 25px;
+    }
+    
+    label {
+        color: ghostwhite;
+        font-size: 30px;
+    }
+    .container {
```

Comment:
As well as formatting, keeping an empty line between CSS rules also makes it easier for us to read our stylesheets. For example, line 22 would be empty and .container would be on line 23.